calculator/operations.py

```diff
@@ -1,19 +1,21 @@
+from functools import reduce
 
 def add(*args):
     # your implementation here
-    pass
+    return reduce(lambda x, y: x + y, args)
 
 def subtract(*args):
     # your implementation here
-    pass
+    return reduce(lambda x, y: x - y, args)
 
 def multiply(*args):
     # your implementation here
-    pass
+    return reduce(lambda x, y: x * y, args)
+    
 
 def divide(*args):
     # your implementation here
-    pass
+    return reduce(lambda x, y: float(x) / y, args)
```

Comment:
This was a very clean way to do your operations functions. Wish we would've thought of this when we did this part